### PR TITLE
Factories have been changed to generate unique data for unique fields…

### PR DESCRIPTION
**Pull request by AndrewTehan**
# Description
There were errors in the seed file caused by Facker generating non-unique data. Bugs fixed

## Type of change

- [x] Bug fix
- [ ] New feature 

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by PandaGrom — APPROVED** [review]

**Review by alhimick-prog — APPROVED** [review]

File: config/environments/development.rb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 require "active_support/core_ext/integer/time"
+require 'factory_bot_rails'
```
Comment (nieisnuje) [inline config/environments/development.rb:2]: will your seeds work on Heroku if you require this only for development and test envs?

Comment (AndrewTehan) [reply config/environments/development.rb:2]: I think no, but I can push my db on Heroku `heroku pg:push ...` and we will have our data there . Should I redo this for a production environment?

Comment (nieisnuje) [reply config/environments/development.rb:2]: yes. you can just require it in application.rb
it's not a good idea to push your db to heroku. it's better to have an empty db there and run seeds in heroku console

Comment (AndrewTehan) [reply config/environments/development.rb:2]: done

**Review by nieisnuje — APPROVED** [review]

**Pull request merged**